libraries/dagster-teradata/dagster_teradata/resources.py

```diff
@@ -85,6 +117,38 @@ def get_connection(self):
             connection_params["database"] = self.database
         if self.logmech is not None:
             connection_params["logmech"] = self.logmech
+        if self.http_proxy is not None:
+            connection_params["http_proxy"] = self.http_proxy
+        if self.http_proxy_user is not None:
+            connection_params["http_proxy_user"] = self.http_proxy_user
+        if self.http_proxy_password is not None:
+            connection_params["http_proxy_password"] = self.http_proxy_password
+        if self.https_proxy is not None:
+            connection_params["https_proxy"] = self.https_proxy
+        if self.https_proxy_user is not None:
+            connection_params["https_proxy_user"] = self.https_proxy_user
+        if self.https_proxy_password is not None:
+            connection_params["https_proxy_password"] = self.https_proxy_password
+        if self.proxy_bypass_hosts is not None:
+            connection_params["proxy_bypass_hosts"] = self.proxy_bypass_hosts
+        if self.sslmode is not None:
+            connection_params["sslmode"] = self.sslmode
```

Comment:
Mohan if you are specifying sslmode, there are two options verify-ca and verify-full. These requires ca file. for reference, please check https://github.com/Teradata/airflow/blob/d74dd4112ffb76b706b73f608305ef50fb16d7df/providers/teradata/src/airflow/providers/teradata/hooks/teradata.py#L177